lib/popup.ts

```diff
@@ -158,6 +158,16 @@ export class PopupControl<T extends IPopupOptions = IPopupOptions> extends Dispo
   private _showDelay: number = 0;
   private _hideDelay: number = 0;
 
+  constructor() {
+    super();
+    // This prevents a bug from happening in Grist, where holding the enter key down on a Choice
```

Comment:
Mmm, trying to think how to make this comment more useful for the future. How about "Clear timeouts on disposal, which might still be scheduled and cause JS errors in case of quick open/close/open interactions" ?